cmd/api/auth/authz.go

```diff
@@ -36,44 +37,59 @@ func RBACAuthorization(sari clientAuthzv1.SubjectAccessReviewInterface, cache *c
 			verb = "get"
 		}
 
-		sarSpec := apiAuthzv1.SubjectAccessReviewSpec{
-			User:   userInfo.Username,
-			Groups: userInfo.Groups,
-			ResourceAttributes: &apiAuthzv1.ResourceAttributes{
-				Namespace: c.Param("namespace"),
-				Group:     c.Param("group"),
-				Version:   c.Param("version"),
-				Resource:  c.Param("resourceType"),
-				Name:      c.Param("name"),
-				Verb:      verb,
-			},
-		}
+		sarSpecs := []apiAuthzv1.SubjectAccessReviewSpec{
+			{
+				User:   userInfo.Username,
+				Groups: userInfo.Groups,
+				ResourceAttributes: &apiAuthzv1.ResourceAttributes{
+					Namespace: c.Param("namespace"),
+					Group:     c.Param("group"),
+					Version:   c.Param("version"),
+					Resource:  c.Param("resourceType"),
+					Name:      c.Param("name"),
+					Verb:      verb,
+				},
+			}}
 
-		allowed := cache.Get(sarSpec.String())
-		if allowed != nil {
-			if allowed != true {
-				abort.Abort(c, "Unauthorized", http.StatusUnauthorized)
-				return
+		if strings.HasSuffix(c.Request.RequestURI, "log") {
```

Comment:
```suggestion
		if strings.HasSuffix(c.Request.URL.Path, "/log") {
```

The `http.Request` docs recommend not using `.RequestURI` but `.URL` instead. Anyway, I also suggest you add a leading `/` so this don't trigger on resources that end with `log`. I don't think there will be many, but it can happen.

Reply:
good point! Changes applied